src/react/components/_academics/EnrollmentNotice.js

```diff
@@ -5,39 +5,49 @@ import { react2angular } from 'react2angular';
 
 import ReduxProvider from 'react/components/ReduxProvider';
 
-import styles from './COVIDEnrollmentNotice.module.scss';
+import styles from './EnrollmentNotice.module.scss';
 import 'icons/bullhorn-solid.svg';
 
 const propTypes = {
   termId: PropTypes.string.isRequired,
   enrollmentTerms: PropTypes.array,
 };
 
-const COVIDEnrollmentNotice = ({ termId, enrollmentTerms }) => {
-  const threshold = 140;
+function renderMessage(preview, more, expanded) {
+  if (expanded) {
+    return [preview, more].join('');
+  } else {
+    return preview;
+  }
+}
+
+const EnrollmentNotice = ({ termId, enrollmentTerms }) => {
+  // if found in the message <read-more>, the component will show everything
+  // before that message, and show a "Read more" button
+  const cutDelimiter = '<read-more>';
   const [expanded, setExpanded] = useState(false);
+  const [preview, more] = message.split(cutDelimiter);
```

Comment:
@developish - Does this mean that the text could be provided from some data source, like a Campus Solutions message catalog entry, and that the HTML/text string that contains '<read-more>' would still be used to cutoff the text?

Reply:
Exactly, and the message catalog is the data source in this instance